language-configuration.json

```diff
@@ -1,57 +1,46 @@
 {
     "comments": {
         "lineComment": "--",
-        "blockComment": [
-            "{-",
-            "-}"
-        ]
+        "blockComment": ["{-", "-}"]
     },
     "brackets": [
-        [
-            "{",
-            "}"
-        ],
-        [
-            "(",
-            ")"
-        ]
+        ["(", ")"]
+        ["{", "}"],
+        ["{!", "!}"],
+        ["{-#", "#-}"],
     ],
     "autoClosingPairs": [
         {
-            "open": "{",
-            "close": "}"
+          "open": "\"",
+          "close": "\"",
+          "notIn": ["comment", "string"]
         },
         {
-            "open": "(",
-            "close": ")"
+          "open": "{!",
+          "close": "!}",
+          "notIn": ["comment", "string"]
         },
         {
-            "open": "\"",
-            "close": "\"",
-            "notIn": [
-                "string"
-            ]
+          "open": "{-#",
+          "close": "#-}",
+          "notIn": ["comment", "string"]
+            
         },
         {
-            "open": "{-",
-            "close": "-}",
-            "notIn": [
-                "comment"
-            ]
+          "open": "⦃",
+          "close": "⦄",
+          "notIn": ["comment", "string"]
```

Comment:
The issue with this is that it will interfere with agda-input. I.e. if you are trying to input `⦃` using agda-input, it will interrupt and insert `{{}}` instead.

Reply:
Thanks for the clarification.